### DISCUSSION TITLE
Wrong battery percentage.

### DISCUSSION THREAD

**Discussion by KoalaV2** [Alexays/Waybar#1827] [Q&A]
The builtin battery module appears to be showing very very wrong battery readings, even showing different on different monitors.

Version: `Waybar v0.9.7-107-g1b4ddbc (branch 'master')`

![image](https://user-images.githubusercontent.com/27083886/126868765-d9025597-3553-4080-93fc-1a7b14040f8d.png)

( Example in the image where it shows 15% on left monitor, 6% on right monitor and 11% ( proper reading ) in ACPI )

**Comment by Alexays** [comment]
Is ok with latest master change?

**Comment by KoalaV2** [comment]
I belive so, I haven't tested it much more but it seems to be okay now, especially since i specified the battery in the config file.
